src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java

```diff
@@ -2,150 +2,150 @@
 
 import java.util.List;
 import java.util.Random;
+
 import jp.kusumotolab.kgenprog.ga.variant.Variant;
 import jp.kusumotolab.kgenprog.ga.variant.VariantStore;
 
 /**
  * 交叉を表すインターフェース．<br>
  * 交叉アルゴリズムを実装するクラスはこのインターフェースを実装しなければならない．<br>
- * 
- * @author higo
  *
+ * @author higo
  */
 public interface Crossover {
 
   /**
    * 交叉処理を行うメソッド．<br>
    * 交叉対象の個体群を含んだVariantStoreを引数として与える必要あり．<br>
-   * 
+   *
    * @param variantStore 交叉対象の個体群
    * @return 交叉により生成された個体群
    */
   List<Variant> exec(VariantStore variantStore);
 
   /**
    * 1つ目の親を返す．
-   * 
+   *
    * @return 1つ目の親
    */
   FirstVariantSelectionStrategy getFirstVariantSelectionStrategy();
 
   /**
    * 2つ目の親を返す．
-   * 
+   *
    * @return 2つ目の親
    */
   SecondVariantSelectionStrategy getSecondVariantSelectionStrategy();
 
   /**
    * kGenProgが基本実装として持つ交叉種別を表す列挙型．
-   * 
+   *
    * TODO ここに定義すべきではないものな気がするので将来移動させる予定．
-   * 
-   * @author higo
    *
+   * @author higo
    */
   enum Type {
 
     /**
      * ランダム交叉を表す型．
-     * 
+     *
      * @see jp.kusumotolab.kgenprog.ga.crossover.RandomCrossover
      */
     Random {
-
       /**
        * 交叉を行うインスタンスを生成するメソッド．
-       * 
+       *
        * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
-       *      SecondVariantSelectionStrategy, int)
-       * 
+       *      SecondVariantSelectionStrategy, int, boolean)
+       *
        * @param random 交叉処理の内部でランダム処理を行うためのシード
        * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム
        * @param secondVariantSelectionStrategy 2つ目の親を選ぶためのアルゴリズム
        * @param generatingCount 一世代の交叉処理で生成する個体の数
+       * @param needHistoricalElement
```

Comment:
上に倣ってパラメータ `needHistricalElement` の説明を追加しよう
```suggestion
       * @param needHistoricalElement ほげほげ
```

なんやこれと思ったらJavadocの書き方をググってみてください